UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -837,6 +884,10 @@ public void Serialize(UndertaleWriter writer)
             }
         }
 
+        /// <summary>
+        /// Deserializes the object from a specified <see cref="UndertaleReader"/>.
+        /// </summary>
+        /// <param name="reader">Where to deserialize from.</param>
```

Comment:
This documentation is incomplete: description is not clear enough what's different betwen `Unserialize(UndertaleReader)` and `uint entryCount` is not documented.